src/snowflake/snowpark/dataframe_writer.py

```diff
@@ -129,6 +129,12 @@ def save_as_table(
         max_data_extension_time: Optional[int] = None,
         change_tracking: Optional[bool] = None,
         copy_grants: bool = False,
+        is_iceberg: bool = False,
```

Comment:
How about the `df.copy_into_table()`?

Reply:
I tested out copy_into_table statements and they work without the iceberg keyword even if you're copying in to an iceberg table, so I don't think we need to update our sql to include it.

Reply:
`copy_into_table` also automatically creates a table if it doesn't exist. So better also add a parameter.

Reply:
Added this functionality.